JobScheduler/ApiRequestHandler.cs

```diff
@@ -75,7 +75,7 @@ public async Task<List<UserTask>> GetExpectedGraduationUsersAsync()
         /// <param name="userTaskId"></param>
         public void SetGraduationTaskStatusToMailed(int userTaskId)
         {
-            client.PutAsync("api/UserTask/SetToMailed/" + userTaskId, null);
+            client.PutAsync("localhost/UserTask/SetToMailed/" + userTaskId, null);
```

Comment:
Why did you change this? I guess it should be api/UserTask/CreateUserTasks/ since it is communicating with the api.

The base url is set in the program of this application.